src/main/java/frc/robot/Robot.java

```diff
@@ -85,7 +95,10 @@ public void disabledInit() {
 
   /** This function is called periodically when the robot is disabled */
   @Override
-  public void disabledPeriodic() {}
+  public void disabledPeriodic() {
+    m_robotContainer.allianceChooserUpdate();
+    m_autoCommand = m_robotContainer.getAutonomousCommand(); // update the auto command before auto starts
```

Comment:
Delete these. They do not need to be done every 20 milliseconds.

The first sets the Alliance. Call the DriverStation method when it is needed.

The autonomous command selection is done (once) in autonomousInit().